lib/livingstyleguide/tilt_template.rb

```diff
@@ -85,6 +85,10 @@ def find_root_path
           break
         end
         path = File.expand_path('..', path)
+        if path === '/'
```

Comment:
Avoid the use of the case equality operator `===`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.